controllers/controllervm_controller.go

```diff
@@ -240,6 +267,39 @@ func (r *ControllerVMReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 	return ctrl.Result{}, nil
 }
 
+func (r *ControllerVMReconciler) setControllerHostStatus(instance *ospdirectorv1beta1.ControllerVM, hostname string, ipaddress string) {
```

Comment:
Just realized that I added this function to set the ControllerHostStatus but I'm not actually calling it anywhere yet! On it...